config/dex/config.yaml

```diff
@@ -1,9 +1,6 @@
 issuer: http://127.0.0.1:5556
 storage:
-  config:
-    file: /tmp/dex.db
-    inCluster: true
-  type: sqlite3
+  type: memory
 logger:
   level: debug
```

Comment:
is log level `debug` as default a good idea? Is it ensured nothing critical is logged?

Reply:
Same here, just for the testcases, so debugging is fine here.